ext/session/tests/session_set_save_handler_closures.phpt

```diff
@@ -41,7 +42,12 @@ $_SESSION['Bar'] = 'Foo';
 var_dump($_SESSION);
 session_write_close();
 
+echo "Cleanup\n";
+session_start();
+session_destroy();
+
 ob_end_flush();
+@rmdir($path);
```

Comment:
Should go in the CLEAN section

Reply:
yeah, it should, but it wouldn't work well on its own, unless the save path and the session handlers are set again, if I understand it correctly.

Reply:
Welp, not a big deal